test/core/extensions/coreIssuanceOrder.spec.ts

```diff
@@ -212,6 +225,26 @@ contract('CoreIssuanceOrder', accounts => {
         zeroExOrderTakerAssetAmount,                      // amount of zeroExOrder to fill
       );
 
+      // Create Kyber trade for the third component, using ether(25) makerToken. Conversion rate pre set on snapshot
+      const sourceTokenQuantity = ether(25);
+      const maxDestinationQuantity = issuanceOrderRequiredComponentAmounts[2];
+      const componentTokenDecimals = (await thirdComponent.decimals.callAsync()).toNumber();
+      const sourceTokenDecimals = (await makerToken.decimals.callAsync()).toNumber();
+      kyberConversionRatePower = new BigNumber(10).pow(18 + sourceTokenDecimals - componentTokenDecimals);
+      const minimumConversionRate = maxDestinationQuantity.div(sourceTokenQuantity)
+                                                          .mul(kyberConversionRatePower)
+                                                          .round();
+      kyberTradeMakerTokenChange = sourceTokenQuantity.sub(
+        maxDestinationQuantity.mul(kyberConversionRatePower).div(KYBER_RESERVE_CONFIGURED_RATE).floor());
```

Comment:
Possible to separate math steps into their own line here?

Reply:
Not a lot of clarity or meaning for the in between steps in this case

Reply:
Oh, I see what you mean, I thought it was confusing cause it's layered inside one sub already so I didn't do that here